client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/RequestStats.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Box } from '@openmsupply-client/common';
+import { DraftRequestLine } from '../hooks';
+import { StockDistribution } from './StockDistribution';
+import { ConsumptionHistory } from './ConsumptionHistory';
+import { StockEvolution } from './StockEvolution';
+
+export interface RequestStatsProps {
+  draft?: DraftRequestLine | null;
+}
+
+export const RequestStats = ({ draft }: RequestStatsProps) => {
+  return (
+    <Box
+      sx={{
+        minHeight: 200,
+        maxHeight: 500,
+        width: 500,
+      }}
+    >
+      {!!draft && (
+        <StockDistribution
+          availableStockOnHand={draft?.itemStats?.availableStockOnHand}
+          averageMonthlyConsumption={
+            draft?.itemStats?.averageMonthlyConsumption
+          }
+          suggestedQuantity={draft?.suggestedQuantity}
+        />
```

Comment:
I was also wondering if it would be possible to hide some graphs if data is not available so you wouldn't need to scroll.

In this case, might be nicer to not show the graph?

![Screenshot 2024-12-09 at 8 47 33 AM](https://github.com/user-attachments/assets/df53eb0f-22c7-4f95-a4de-f69c451c74cb)

I'm not sure if this is a niche case, and might not be worth worrying about